be/src/cloud/cloud_schema_change_job.cpp

```diff
@@ -541,6 +542,9 @@ Status CloudSchemaChangeJob::_process_delete_bitmap(int64_t alter_version,
 }
 
 void CloudSchemaChangeJob::clean_up_on_failed() {
+    _cloud_storage_engine.meta_mgr().remove_delete_bitmap_update_lock(
```

Comment:
call remove_delete_bitmap_update_lock only when this is a mow tablet

Reply:
+1